src/EditorFeatures/Core/Implementation/ChangeSignature/AbstractChangeSignatureCommandHandler.cs

```diff
@@ -28,34 +28,25 @@ public VSCommanding.CommandState GetCommandState(ReorderParametersCommandArgs ar
         public VSCommanding.CommandState GetCommandState(RemoveParametersCommandArgs args)
             => GetCommandState(args.SubjectBuffer);
 
-        private static VSCommanding.CommandState GetCommandState(ITextBuffer subjectBuffer)
-        {
-            return IsAvailable(subjectBuffer) ? VSCommanding.CommandState.Available : VSCommanding.CommandState.Unspecified;
-        }
+        private static VSCommanding.CommandState GetCommandState(ITextBuffer subjectBuffer) =>
+            IsAvailable(subjectBuffer, out _) ? VSCommanding.CommandState.Available : VSCommanding.CommandState.Unspecified;
```

Comment:
nit: IDE style is `=>` on the next line when things are wrapped.  You can see that below :)

Reply:
right you are, old habits die hard

Reply:
Tell me about it!  I have to flip between multiple codebases with different curly brace placement.  I have constant brainfarts.   And i won't even mention how often i write "const" in C# :D